lib/typings/operations/catalogItems.ts

```diff
@@ -1,15 +1,17 @@
 import { BaseResponse } from "../baseTypes";
 
 export interface GetCatalogItemQuery {
-  MarketplaceId: string;
+  includedData?: string;
+  locale?: string;
+  marketplaceIds: string[];
 }
 
 export interface GetCatalogItemPath {
   asin: string;
 }
 
 export interface GetCatalogItemResponse extends BaseResponse {
-  payload?: ListCatalogItem;
+  payload?: any;
```

Comment:
This is a more correct type than `ListCatalogItem`

Reply:
Good to have GetCatalogItemQuery updated to the new structure, the previous one was still from the deprecated v0 endpoint.

Reply:
can this be punted to a separate issue? i was going to add it but the typings are really large for this

Reply:
scope of this PR is already large

Reply:
Yes, I think thats fine. The TS part would need some more major updates anyway, things that have been introduced in the meantime (i.e. endpoint param etc.) are also not included yet.

Reply:
I'm not sure what the ask is here